content/tracing/advanced_usage/_index.md

```diff
@@ -279,6 +279,88 @@ if (($span = GlobalTracer::get()->getActiveSpan()) !== null){
 {{% /tab %}}
 {{< /tabs >}}
 
+## Change Agent Hostname
+
+Configure your application level tracers to submit traces to a custom Agent hostname. See the examples below for each supported language:
+
+{{< tabs >}}
+{{% tab "Java" %}}
+
+The Java Tracing Module automatically looks for and initializes with the ENV variables DD_AGENT_HOST and DD_TRACE_AGENT_PORT:
```

Comment:
This must be replicated for all languages:

The <LANGUAGE> Tracing Module automatically looks for and initializes with the ENV variables `DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT`

Reply:
(also backtick are missing in DD_AGENT_HOST and DD_TRACE_AGENT_PORT)